scipy/linalg/basic.py

```diff
@@ -551,7 +552,7 @@ def solve_circulant(c, b, singular='raise', tol=None,
 
     Check by solving one pair of `c` and `b` vectors (cf. ``x[1, 1, :]``):
 
-    >>> solve_circulant(c[1], b[:,1])
+    >>> solve_circulant(c[1], b[1, :])
```

Comment:
This seems to be just a typo left over from API tweaks. The original version throws a ValueError, the tweaked version works and seems to make sense.